utcaApp/src/afc_timing.cpp

```diff
@@ -33,50 +24,56 @@ class AFCTiming: public UDriver {
+              {"LOCKED_GT0", p_read_only},
+              {"LOCKED_AFC_FREQ_LTC", p_read_only},
+              {"LOCKED_AFC_PHASE_LTC", p_read_only},
+              {"LOCKED_RTM_FREQ_LTC", p_read_only},
+              {"LOCKED_RTM_PHASE_LTC", p_read_only},
+              {"LOCKED_GT0_LTC", p_last_lock},
+              {"ALIVE", p_read_only},
+              ParamInit{"RST_LOCKED_LTCS", p_decoder_controller}.set_wo(),
           },
           {
-              ParamInit{"RFREQ_HI", p_rfreq_hi}.set_nc(2),
-              ParamInit{"RFREQ_LO", p_rfreq_lo}.set_nc(2),
-              ParamInit{"N1", p_n1}.set_nc(2),
-              ParamInit{"HS_DIV", p_hs_div}.set_nc(2),
-              ParamInit{"FREQ_KP", p_freq_kp}.set_nc(2),
-              ParamInit{"FREQ_KI", p_freq_ki}.set_nc(2),
-              ParamInit{"PHASE_KP", p_phase_kp}.set_nc(2),
-              ParamInit{"PHASE_KI", p_phase_ki}.set_nc(2),
-              ParamInit{"MAF_NAVG", p_maf_navg}.set_nc(2),
-              ParamInit{"MAF_DIV_EXP", p_maf_divexp}.set_nc(2),
-
-              {"CH_EN", p_ch_en},
-              {"CH_POL", p_ch_pol},
-              {"CH_LOG", p_ch_log},
-              {"CH_ITL", p_ch_itl},
+              ParamInit{"RFREQ_HI", p_read_only}.set_nc(2),
+              ParamInit{"RFREQ_LO", p_read_only}.set_nc(2),
+              ParamInit{"N1", p_read_only}.set_nc(2),
+              ParamInit{"HS_DIV", p_read_only}.set_nc(2),
+              ParamInit{"FREQ_KP", p_decoder_controller}.set_nc(2),
+              ParamInit{"FREQ_KI", p_decoder_controller}.set_nc(2),
+              ParamInit{"PHASE_KP", p_decoder_controller}.set_nc(2),
+              ParamInit{"PHASE_KI", p_decoder_controller}.set_nc(2),
+              ParamInit{"MAF_NAVG", p_decoder_controller}.set_nc(2),
+              ParamInit{"MAF_DIV_EXP", p_decoder_controller}.set_nc(2),
+
+              {"CH_EN", p_decoder_controller},
+              {"CH_POL", p_decoder_controller},
+              {"CH_LOG", p_decoder_controller},
+              {"CH_ITL", p_decoder_controller},
               {"CH_SRC", p_ch_src},
-              {"CH_DIR", p_ch_dir},
-              {"CH_PULSES", p_ch_pulses},
-              ParamInit{"CH_COUNT_RST", p_ch_count_rst}.set_wo(),
-              {"CH_COUNT", p_ch_count},
-              {"CH_EVT", p_ch_evt},
-              {"CH_DLY", p_ch_dly},
-              {"CH_WDT", p_ch_wdt},
-          }),
+              {"CH_DIR", p_decoder_controller},
+              {"CH_PULSES", p_decoder_controller},
+              ParamInit{"CH_COUNT_RST", p_decoder_controller}.set_wo(),
+              {"CH_COUNT", p_read_only},
+              {"CH_EVT", p_decoder_controller},
+              {"CH_DLY", p_decoder_controller},
+              {"CH_WDT", p_decoder_controller},
+          }, &ctl),
       dec(bars),
       ctl(bars)
     {
         auto v = find_device(port_number);
         dec.set_devinfo(v);
         ctl.set_devinfo(v);
 
+        parameter_props.at(p_ch_src).write_decoder_controller = true;
+
```

Comment:
This seems a little bit hackish. We only avoid creating it with `p_decoder_controller` counter to later properly identify and treat it on `readEnum`, right?

I feels like defining a parameter as user of decoder controller should be disassociated with such counter. Don't have a concrete proposal, though. Just make it clear with a comment why such hack is interesting for now.

Reply:
`p_decoder_controller` is not used as a counter; its main purpose is simply being a variable with a known address.

Maybe I should have another way of specifying that something uses the decodercontroller that doesn't involve the param numbers :thinking: 

I will add a comment for it, though I would note that it's already been done in another module.

Reply:
> its main purpose is simply being a variable with a known address

That's true. I was thinking more about how its content turn out to be filled by `createParam`.

> I will add a comment for it, though I would note that it's already been done in another module.

I indeed didn't notice this was already done. Well, feel free not to do so, and rather let this pending if we turn out changing how to specify this.